composer.json

```diff
@@ -19,17 +19,17 @@
     ],
     "license": "MIT",
     "require": {
-        "illuminate/support": "^9.0",
-        "illuminate/container": "^9.0",
-        "illuminate/database": "^9.0",
-        "illuminate/events": "^9.0",
-        "mongodb/mongodb": "^1.11"
+        "illuminate/support": "^10.0",
+        "illuminate/container": "^10.0",
+        "illuminate/database": "^10.0",
+        "illuminate/events": "^10.0",
+        "mongodb/mongodb": "^1.15"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5.8",
-        "orchestra/testbench": "^7.0",
-        "mockery/mockery": "^1.3.1",
-        "doctrine/dbal": "^2.13.3|^3.1.4"
+        "phpunit/phpunit": "^9.5.10",
+        "orchestra/testbench": "^8.0",
+        "mockery/mockery": "^1.4.4",
+        "doctrine/dbal": "^3.5"
```

Comment:
Note to self: we should try to get rid of this dependency. With Laravel itself no longer depending on it, we should be able to drop the dependency as well at some point.

Reply:
We need to get rid of a lot things, library needs a complete revamp even though the new version of Laravel comes soon.